### PR TITLE
#709 do not clear input value in multiselect if prop `onSelectResetsInput={false}` is passed

### PR DESCRIPTION
**Pull request by nosovsh**
I can add tests and update readme and example if you are going to accept this

### CODE REVIEW

File: src/Select.js [hunk 1]
```diff
@@ -392,7 +394,7 @@ const Select = React.createClass({
 		if (this.props.multi) {
 			this.addValue(value);
 			this.setState({
-				inputValue: '',
+				inputValue: this.props.onSelectResetsInput ? '' : this.state.inputValue,
```
Comment (wmertens) [inline src/Select.js]: When you're leaving the value as-is, there is no reason for the setState, so better put it in an `if` around it

File: src/Select.js [hunk 2]
```diff
@@ -571,8 +573,8 @@ const Select = React.createClass({
 	renderClear () {
 		if (!this.props.clearable || !this.props.value || (this.props.multi && !this.props.value.length) || this.props.disabled || this.props.isLoading) return;
 		return (
-			<span className="Select-clear-zone" title={this.props.multi ? this.props.clearAllText : this.props.clearValueText} 
-						aria-label={this.props.multi ? this.props.clearAllText : this.props.clearValueText} 
+			<span className="Select-clear-zone" title={this.props.multi ? this.props.clearAllText : this.props.clearValueText}
+						aria-label={this.props.multi ? this.props.clearAllText : this.props.clearValueText}
```
Comment (wmertens) [inline src/Select.js]: I don't know @JedWatson's preferences, but is seems to me that lint-type changes should go in a separate commit…

File: src/Select.js (hunk 1 above)
Comment (trashgenerator) [inline src/Select.js]: fair enough. Changed in the next commit

File: src/Select.js (hunk 2 above)
Comment (trashgenerator) [inline src/Select.js]: hmm. seems it was added by IDE. Is it possible to remove it from the commit after push?

**Comment by JedWatson** [comment]
Implemented by #1952
